02.calendar/calendar.rb

```diff
@@ -0,0 +1,54 @@
+require 'optparse'
+require 'date'
+params = ARGV.getopts("m:","y:")
+
+month = params["m"].to_i
+year = params["y"].to_i
+
+def check_month(month)
+  if month == 0
+    return Date.today.month
+  else
+    return month
+  end
```

Comment:
Rubyでは最後の評価された値が返り値になるため、`return`は必要ありません。
```suggestion
  if month == 0
    Date.today.month
  else
    month
  end
```

また、`if`と`else`の中身が両方一行しかないので、三項演算子を使って書くこともできそうですね。好みの問題もあるので以下は必須というわけではありませんが参考にしてください。

```ruby
  month == 0 ? Date.today.month : month
```

個人的には「monthが0でなければmonthを返し、0だった場合は今日の月を返す」という順番で考えた方が分かりやすいので、下記のようにした方が理解しやすいように思います。

```ruby
  month != 0 ? month : Date.today.month
```

また、`Integer`クラスの`nonzero?`を使えばこんな感じでも書けます。

```ruby
  month.nonzero? || Date.today.month
```

参考：https://docs.ruby-lang.org/ja/latest/method/Numeric/i/nonzero=3f.html